e2e/node/e2e.test.ts

```diff
@@ -228,29 +236,11 @@ describe(`End-to-end access grant tests for environment [${environment}] `, () =
 
   beforeAll(async () => {
     // Log both sessions in.
-    // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    allSessions = [
-      retryAsync(getRequestorSession),
-      retryAsync(() => getAuthenticatedSession(env)),
-    ];
-    sessionInterval = setInterval(
-      async () => {
-        const nextSessions = [
-          retryAsync(getRequestorSession),
-          retryAsync(() => getAuthenticatedSession(env)),
-        ];
-        allSessions.push(...nextSessions);
-        [requestorSession, resourceOwnerSession] =
-          await Promise.all(nextSessions);
-      },
-      4 * 60 * 1000,
-    );
-
-    [requestorSession, resourceOwnerSession] = await Promise.all(allSessions);
```

Comment:
This is simplified using two session objects initialized once and with a listener on the expiration event to log the session back in.